### DISCUSSION TITLE
Can not install whisper because error: subprocess-exited-with-error

### DISCUSSION THREAD

**Discussion by artrayd** [openai/whisper#158] [Q&A] [answered]
I run this command
` pip install git+https://github.com/openai/whisper.git `

And get this error:
```
error: subprocess-exited-with-error

× git version did not run successfully.
│ exit code: 72
╰─> See above for output.
```

On MacOS Monterey

**Comment by jongwook** [comment, marked as answer]
Do you have git installed, and can you run

```bash
git clone https://github.com/openai/whisper.git
```

from your terminal?

**Comment by artrayd** [comment]
Thank you, that was the problem! Git wasn't installed on my comp. I had github desktop - but somehow didn't had git.
